util/helper.go

```diff
@@ -323,3 +329,10 @@ func MatchRegexExpression(exp string, text string) (bool, error) {
 	matched := rExp.Match([]byte(text))
 	return matched, nil
 }
+
+func GetLatestImageAccToImagePushedAt(imageDetails []types.ImageDetail) types.ImageDetail {
+	sort.Slice(imageDetails, func(i, j int) bool {
+		return imageDetails[i].ImagePushedAt.After(*imageDetails[j].ImagePushedAt)
+	})
+	return imageDetails[0]
```

Comment:
what if imageDetails nil or empty

Reply:
Have Already handled it where it is  used.